.env

```diff
@@ -21,6 +21,10 @@ REACT_APP_JLAB_FALLBACK_DOMAIN="water_224.h5"
 # Rendering performance profiling (local development only)
 REACT_APP_PROFILING_ENABLED=false
 
+# Disable ESLint error modal in development (log warnings instead)
+ESLINT_NO_DEV_ERRORS=true
+
 # Logging limit of React Testing Library
-# https://testing-library.com/docs/dom-testing-library/api-helpers/#debugging
+# Don't override this variable in `.env.local`, as this file is not used during tests
```

Comment:
This means it should be set in `.env` to have an effect ?

Reply:
You can set it temporarily in `.env` and not commit it, on the terminal directly (before `npm test`), or in `.env.test.local` if you want to keep it around.